website/src/components/MiniBlog/index.tsx

```diff
@@ -146,6 +141,24 @@ const MiniBlog: React.FC = () => {
     return tabs;
   });
 
+  useEffect(() => {
+    let tagsArray: Array<string> = [];
+    if (jsonMdData && jsonMdData.constructor === Array) { // this check is necessary to avoid parsing errors on JSON data
+      for (let i=0; i < jsonMdData.length; i++) {
+        tagsArray = [...tagsArray, ...jsonMdData[i].tags];
+      }
+      setTagsDistribution(tagsArray.reduce((acum: any,cur: string) => Object.assign(acum,{[cur]: (acum[cur] || 0)+1}),{}));
+    }
+  }, [jsonMdData]);
+
+  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => 
+          <StyledTab
+            label={item + "(" + tagsDistribution[item as keyof typeof tagsDistribution] + ")"}
```

Comment:
```
tagsDistribution[item as keyof typeof tagsDistribution]
```
this will always return `1`, right??

Reply:
No, it will return the count of the particular tag.

Reply:
object[key] = value; we are trying to extract the count from the key.

Reply:
I assumed that tags are different 😄